python/cudf/dataframe/series.py

```diff
@@ -430,6 +430,8 @@ def _binaryop(self, other, fn):
         """
         from cudf import DataFrame
         if isinstance(other, DataFrame):
+            # i removed _binaryop from dataframe in #1292 (bitwise binary ops)
```

Comment:
```suggestion
            # TODO: removed _binaryop from dataframe in #1292 (bitwise binary ops)
```

Reply:
Is this fix and test in the scope of this PR?

Reply:
It’s not really, I added the comment because I figured out that it was my own change that broke this. It sort of affects this PR also because the exception displayed is wrong in case of a Series (op) DataFrame. It should be a NotImplemented error instead of attribute not found. But now I think this is better as a github issue